common/cosmwasm-smart-contracts/vesting-contract/src/lib.rs

```diff
@@ -42,6 +44,48 @@ impl PledgeData {
     }
 }
 
+#[allow(clippy::derive_partial_eq_without_eq)]
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub enum PledgeCap {
+    Percent(u8),
```

Comment:
I'd just internally keep it as `Decimal` so if somebody has an insane idea of "change the pledge cap to 7.5%" we wouldn't need to do another rework; plus I'd assume all of maths would be easier